src/backend/tokenProxy.ts

```diff
@@ -0,0 +1,69 @@
+import { NextFunction, Request, RequestHandler, Response } from 'express';
```

Comment:
Er dette noe vi (senere) burde flytte til felles? :-)

Reply:
Både ja og nei. Liker ikke å ha "express" i felles-kode. Da er det isåfall selve token-exchange-biten, men synes det er litt hacky å trekke ut avhengigheter som er litt "magiske" og vanskelig å verifisere til felles-kode. 

familie-felles-frontend -> backend er jo "umulig" å videreutvikle ettersom det er avhengigheter på kryss og tvers fra det som er kjørbart